api/src/plugins/sync.py

```diff
@@ -0,0 +1,124 @@
+import json
+
+import requests
+from fastapi.logger import logger
+from src.plugins import PluginInterface
+from src.indicators import update_indicator
+from src.data import get_context
+from validation import IndicatorSchema
+
+
+class SyncPlugin(PluginInterface):
+
+    # Sync annotations to indicators
+    def publish(self, data, type="indicator"):
+        if type == "indicator":
+            uuid = data["id"]
+            context = get_context(uuid)
+
+            # Outputs
+            qualifier_outputs = []
+            outputs = []
+            feature_names = []
+            for feature in context["annotations"]["annotations"]["feature"]:
+
+                feature_names.append(
+                    feature["name"]
+                )  # Used for the primary qualifier outputs.
+                output = dict(
+                    name=feature["name"],
+                    display_name=feature["display_name"],
+                    description=feature["description"],
+                    type=feature["feature_type"],
+                    unit=feature["units"],
+                    unit_description=feature["units_description"],
+                    ontologies={},
+                    is_primary=True,
+                    data_resolution={
+                        "temporal_resolution": context.get("dataset", {}).get(
+                            "temporal_resolution", None
+                        ),
+                        "spatial_resolution": context.get("dataset", {}).get(
+                            "spatial_resolution", None
+                        ),
+                    },
+                    alias=feature["aliases"],
```

Comment:
As aliases are somewhat optional- would it make sense to change this to a `dict.get()` call? I'm worried about users forgetting to add this property when using csv file annotation or other mechanisms.